src/main/resources/assets/indrev/lang/zh_cn.json

```diff
@@ -493,18 +518,34 @@
   "advancements.indrev.buffer_upgrade.description": "获得缓冲升级",
   "advancements.indrev.energy_upgrade": "卓有成效",
   "advancements.indrev.energy_upgrade.description": "获得能效升级",
-  "advancements.indrev.cable_mk1": "橙色电缆！",
-  "advancements.indrev.cable_mk1.description": "获得一根MK1电缆",
-  "advancements.indrev.cable_mk2": "蓝色电缆！",
-  "advancements.indrev.cable_mk2.description": "获得一根MK2电缆",
-  "advancements.indrev.cable_mk3": "紫色电缆！",
-  "advancements.indrev.cable_mk3.description": "获得一根MK3电缆",
-  "advancements.indrev.cable_mk4": "红色电缆！",
-  "advancements.indrev.cable_mk4.description": "获得一根MK4电缆",
+  "advancements.indrev.cable_mk1": "橙色线缆！",
+  "advancements.indrev.cable_mk1.description": "获得一根MK1线缆",
+  "advancements.indrev.cable_mk2": "蓝色线缆！",
+  "advancements.indrev.cable_mk2.description": "获得一根MK2线缆",
+  "advancements.indrev.cable_mk3": "紫色线缆！",
+  "advancements.indrev.cable_mk3.description": "获得一根MK3线缆",
+  "advancements.indrev.cable_mk4": "红色线缆！",
+  "advancements.indrev.cable_mk4.description": "获得一根MK4线缆",
   "advancements.indrev.modular_workbench": "这看起来不像工作台",
   "advancements.indrev.modular_workbench.description": "获得模块化工作台",
   "advancements.indrev.modular_armor": "模块化",
   "advancements.indrev.modular_armor.description": "获得模块化套装",
   "advancements.indrev.gamer_axe": "RGB",
```

Comment:
Is this right? Not sure if it's intended, if it is I'll just merge it. Thank you for the translation!

Reply:
I just change the names.